interface_demo.py

```diff
@@ -0,0 +1,71 @@
+from server import GameServer
+import logging
+from collections import defaultdict
+from ui.user_interface import UserInterface
+
+class InterfaceDemo(GameServer):
+
+    def demo_all_interfaces(self):
+        self.__demo_score_keeper()
+        self.__demo_board()
+        self.__demo_wheel()
+        self.__demo_user_interface()
+        self.__demo_executive_logic()
+
+    def __demo_score_keeper(self):
+        logging.info("\n\nPinging Scorekeeper Interfaces")
+        add_items = [("Wil", 1000, 1), ("Luis", 9999, 3), ("Robyn", 1024, 0)]
+        for player, pts, tokens in add_items:
+            self.score_keeper.playerPoints[player] = pts
+            self.score_keeper.playerTokens[player] = tokens
+        logging.info("get_scores(): %s", self.score_keeper.get_scores())
+        logging.info("get_tokens(): %s", self.score_keeper.get_tokens())
+        logging.info("check_answer(): %s", self.score_keeper.check_answer("Right Answer", "Right Answer", 1, "Luis"))
+        logging.info("has_token('Wil'): %s", self.score_keeper.has_token("Wil"))
+        logging.info("use_token('Wil'): %s", self.score_keeper.use_token("Wil"))
+        logging.info("bankrupt('Robyn'): %s", self.score_keeper.bankrupt("Robyn"))
+
+    def __demo_board(self):
+        logging.info("\n\nPinging Board Interfaces")
+
+        logging.info('get_tile("food"): %s', self.board.get_tile("food", 1))
+        logging.info('is_category_available("politics"): %s', self.board.is_category_available("politics", 1))
+        logging.info("get_available_categories(1): %s", self.board.get_available_categories(1))
+
+    def __demo_wheel(self):
+        logging.info("\n\nPinging Wheel Interfaces")
+
+        logging.info("get_spin_result(): %s", self.wheel.get_spin_result())
```

Comment:
File "/home/lu/Desktop/def_spinit_project/interface_demo.py", line 38, in __demo_wheel
    logging.info("get_spin_result(): %s", self.wheel.get_spin_result())
AttributeError: 'Wheel' object has no attribute 'get_spin_result'. Did you mean: 'get_spint_result'?

Reply:
looks like there a couple of runtime errors, did you push the latest code @Wilson-Orlando

Reply:
I made them match in PR #6 so it's `get_spin_result` in both